src/tests/components/experiments/ExperimentTable.test.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Adapter from "enzyme-adapter-react-16";
+import { shallow, configure } from "enzyme";
+configure({ adapter: new Adapter() });
+import { fromJS } from "immutable";
+import { ExperimentTable } from "../../../components/experiments/ExperimentTable";
+
+describe("ExperimentTable component", () => {
+    const container = shallow(
+        <ExperimentTable 
+          filteredExperiments={fromJS([{name: "Name", start_date: "2018-08-22", end_date: "2018-08-23", status: "live"}])}
+        />
+      );
+    const thead = container.find('thead');
+    const headerRow = thead.find('tr');
+    const tbody = container.find('tbody');
+    const rows = tbody.find('tr');
+
+    it("should render an experiment table with one thead tag", () => {
+        expect(thead).toHaveLength(1);
+    });
+    
+    it("should render an experiment table with correct header rows defining the head of the columns of the table", () => {
+        headerRow.forEach((tr) => {
+            const cells = tr.find('th');
+            expect(cells).toHaveLength(4);
+            expect(cells.at(0).text()).toEqual("Name");
+            expect(cells.at(1).text()).toEqual("Start Date");
+            expect(cells.at(2).text()).toEqual("End Date");
+            expect(cells.at(3).text()).toEqual("Status");
+        });
+    });
+
+    it("should render an experiment table with one tbody tag", () => {
```

Comment:
Usually a single test has three parts

1) set up some condition
2) run some application logic
3) check the results

There are all three parts here, but they're scattered across multiple 'tests', but are they really separate tests?

Reply:
Good point, I modified the tests, see if it's better structured now? :green_apple: